### DISCUSSION TITLE
What is the reference for the oxygen diffusion model?

### DISCUSSION THREAD

**Discussion by RuiheLi** [pybamm-team/PyBaMM#2252] [Q&A] [answered]
Hi PyBaMM team,

I just found that PyBaMM has had a oxygen diffusion submodel, which I am very ineterested of. May I ask what is the reference paper that this submodel based on? Has it integrated with the lithium-ion battery model like DFN or SPM? Big thanks!

**Comment by valentinsulzer** [comment, marked as answer]
It's for lead-acid, reference is [Chapter 5 of my thesis](https://ora.ox.ac.uk/objects/uuid:db582870-4473-411e-8a16-542a8bb0eeca/download_file?file_format=pdf&safe_filename=thesis.pdf&type_of_work=Thesis). Not currently integrated with lithium-ion

**Reply by RuiheLi** [reply]
Thanks a lot!
